resources/charts/bitcoincore/templates/_helpers.tpl

```diff
@@ -55,3 +55,15 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+
+{{/*
+Add network section heading in bitcoin.conf after v0.17.0
+*/}}
+{{- define "bitcoincore.check_semver.regtest" -}}
+{{- $tag := .Values.image.tag | trimPrefix "v" -}}
```

Comment:
we should pick a standard for our tags, either always with v or always without (probs best to follow whatever core is currently doing?), and then re-tag all our images in bitcoindevproject. cc @willcl-ark

Reply:
Yeah it’s bank holiday here today so no work, but I’ll retag all images without v later/tomorrow

Reply:
sounds good, we should also start documenting these conventions _somewhere_, (tho not sure where since this more just for us)

Reply:
Opened #498